pkgs/development/compilers/emscripten/default.nix

```diff
@@ -1,17 +1,17 @@
 { lib, stdenv, fetchFromGitHub, python3, nodejs, closurecompiler
 , jre, binaryen
-, llvmPackages_11
-, symlinkJoin, makeWrapper
+, llvmPackages
+, symlinkJoin, makeWrapper, substituteAll
 , mkYarnModules
 }:
 
 stdenv.mkDerivation rec {
   pname = "emscripten";
-  version = "2.0.10";
+  version = "2.0.26";
 
   llvmEnv = symlinkJoin {
     name = "emscripten-llvm-${version}";
-    paths = with llvmPackages_11; [ clang-unwrapped lld llvm ];
+    paths = with llvmPackages; [ clang-unwrapped clang-unwrapped.lib lld llvm ];
```

Comment:
why remove the pinned version?
If you would prefer upgrading, how about using `_12`
This could create breakage for when people automatically update the llvmPackages attribute.
IMO better to not rely on something that could change.

Reply:
It is still pinned as `llvmPackages = llvmPackages_13;` in `all-packages.nix`...
I don't care where it's pinned, but Sandro requested this change.

Reply:
ok cool, not feeling strongly enough about it.
I was just saying when people update the `llvmPackages` attributes, this might inadertently break this.